### ISSUE TITLE
10 minutes is too long to wait for the first update from console updater

### ISSUE THREAD

**Issue by aserebryakov** [EDSM-NET/FrontEnd#444]
Waiting for 10 minutes before you get the first data update is too much.

It would be great to have a "refresh now" button that can be used once a day.

**Comment by Maddious** [comment]
Yes, I will add to this, that 10mins means we on Xbox need to wait 5 hours per month +/-, if you don't play for a month.
It is a ridiculous long interval, please start using log creation date to scoll ahead instead of 10mins per day!!

**Comment by AnthorNet** [comment]
Working on a better solution ;)

**Comment by Maddious** [comment]
> Working on a better solution ;)

Got back to the game, than tried to update my logs only to find a it went from 3mins to 10mins per day.
I'm sure it has something to do with max requests per min, to the ed's public api. 

You can make dynamic buffer for dymanic refresh times based on the total requests at the moment?

**Comment by aserebryakov** [comment]
> 
> 
> Working on a better solution ;)

Any progress on it?

**Comment by AnthorNet** [comment]
Work has started but not very much yet. 
Waiting for kids to be back at school  ^^

**Comment by AnthorNet** [comment]
Reduced it by half in the meantime :)

**Comment by aserebryakov** [comment]
> 
> 
> Work has started but not very much yet.
> Waiting for kids to be back at school ^^

Perfectly understandable )))

> 
> 
> Reduced it by half in the meantime :)

Thanks a lot! This makes UX much better!
